crates/chainspec/src/api.rs

```diff
@@ -12,9 +12,6 @@ use reth_network_peers::NodeRecord;
 /// Trait representing type configuring a chain spec.
```

Comment:
This file contains the main changes and everything else is downstream of it

Reply:
Note: No changes should be made to the consensus related crates since validation does not need the genesis file